core/string/char_range.inc

```diff
@@ -392,6 +392,7 @@ static CharRange xid_start[] = {
 	{ 0xab5c, 0xab69 },
 	{ 0xab70, 0xabe2 },
 	{ 0xac00, 0xd7a3 },
+	{ 0xb0a1, 0xf7fe },
```

Comment:
We do not use GB2321 encoding anywhere, all strings are UTF-32 or UTF-8. All ranges are UTF-32 ranges.

And for UTF-32 this is definitely wrong, this includes surrogates (`D800—DB7F` and `DC00—DFFF`) which should never be part of any string. And the rest should be already covered by other ranges.

Reply:
I made a mistake... should use unicode 4e00-9fa5.
Actually, CharRange already contains.